deploy/crds/metal3.io_baremetalhosts_crd.yaml

```diff
@@ -470,7 +470,6 @@ spec:
             operationalStatus:
               description: OperationalStatus holds the status of the host
               enum:
-              - ""
```

Comment:
This is very much required otherwise the operator can never update the CR (I discovered this the hard way).

Reply:
OK. Just added it. Please note that format is slight different now with ; as separators.